node/builder.go

```diff
@@ -121,6 +121,7 @@ const (
 	SettlePaymentChannelsKey
 	RunPeerTaggerKey
 	SetupFallbackBlockstoresKey
+	RPCServer
```

Comment:
I'd be specific so that this isn't confused as having anything to do with JsonRPC

```suggestion
	GoRPCServer
```

Reply:
done